src/AppInstallerCLICore/COMContext.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+#include "pch.h"
+#include "..\AppInstallerCommonCore\Public\AppInstallerProgress.h"
+#include "ExecutionContext.h"
+
+namespace AppInstaller
+{
+	using namespace AppInstaller::CLI;

```

Comment:
>	 [](start = 0, length = 1)

Set your editor to use 4 spaces rather than tabs.

Reply:
Still see tabs in this file.

Reply:
missed them. thanks for spotting.